core/templates/dev/head/pages/exploration_editor/statistics_tab/IssuesDirective.js

```diff
@@ -60,6 +60,13 @@ oppia.directive('issuesDirective', [
               'issueNav' + nextIssueIndex).classList.add('bg-clr');
             $scope.currentIssueIndex = nextIssueIndex;
           };
+
+          $scope.resolveIssue = function(currentIssueIndex) {
+            var issue = $scope.issues[currentIssueIndex];
+            IssuesService.resolveIssue(issue);
+            // TODO(pranavsid98): Figure out how to do a hard refresh.
```

Comment:
Explain what will happen without a hard refresh, and why we want one?

Reply:
@brianrodri This is something I want to check for myself after the three issue directives go in. I wanna see if a hard refresh would actually be required to update the issues nav column to handle the deleted issue.